Assets/Scripts/SS3D/Systems/Tile/PlacedObjects/PlacedTileObject.cs

```diff
@@ -0,0 +1,145 @@
+using FishNet;
+using FishNet.Object;
+using SS3D.Core;
+using SS3D.Logging;
+using SS3D.Systems.Tile.Connections;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SS3D.Systems.Tile
+{
+    public class PlacedTileObject: NetworkBehaviour
```

Comment:
Make a summary to explain what this is (and especially why we need that different from placedItemObject)